packages/flutter/lib/src/material/slider.dart

```diff
@@ -536,7 +537,18 @@ class _RenderSlider extends RenderBox {
   static const double _preferredTrackWidth = 144.0;
   static const double _preferredTotalWidth = _preferredTrackWidth + _overlayDiameter;
   static const Duration _minimumInteractionTime = const Duration(milliseconds: 500);
-  static const double _adjustmentUnit = 0.1; // Matches iOS implementation of material slider.
+  static double get _adjustmentUnit {
+    switch (defaultTargetPlatform) {
```

Comment:
you should basically never use defaultTargetPlatform. Always get the platform from the theme.

Reply:
Fixed